accounts-db/src/accounts.rs

```diff
@@ -81,12 +81,14 @@ impl Accounts {
         }
     }
 
+    /// Return loaded addresses and the deactivation slot.
+    /// If no tables are de-activating, the deactivation slot is `u64::MAX`.
```

Comment:
nit: this function only looks up addresses for one ALT
```suggestion
    /// If the table hasn't been deactivated, the deactivation slot is `u64::MAX`.
```